src/styles/utils/calendar-modification.scss

```diff
@@ -0,0 +1,27 @@
+@for $day from 1 through 31 {
+  .calendar__day:nth-child(#{$day})::before {
+    content: "#{$day}";
+  }
+}
+
+@each $day in mon, tue, wed, thu, fri, sat, sun {
+  .calendar--start-day-#{$day} {
+    .calendar__day:first-child {
+      margin-left: if($day == mon, 0,
+      if($day == tue, $cell-size + 1,
+      if($day == wed, ($cell-size * 2) + 2,
+      if($day == thu, ($cell-size * 3) + 3,
+      if($day == fri, ($cell-size * 4) + 4,
+      if($day == sat, ($cell-size * 5) + 5,
+      $cell-size * 6 + 6))))));
+    }
+  }
+}
+
+@each $length in 28, 29, 30, 31 {
```

Comment:
Use `for` here
![image](https://github.com/mate-academy/layout_calendar/assets/96409563/78f63721-63bb-467f-b144-75a6989a5e11)